class.json-api-endpoints.php

```diff
@@ -1084,14 +1101,16 @@ function get_media_item_v1_1( $media_id ) {
 
 		if ( in_array( $ext, array( 'mp3', 'm4a', 'wav', 'ogg' ) ) ) {
 			$metadata = wp_get_attachment_metadata( $media_item->ID );
+			$response['length'] = $metadata['length'];
 			$response['exif']   = $metadata;
 		}
 
 		if ( in_array( $ext, array( 'ogv', 'mp4', 'mov', 'wmv', 'avi', 'mpg', '3gp', '3g2', 'm4v' ) ) ) {
 			$metadata = wp_get_attachment_metadata( $media_item->ID );
-			if ( isset( $metadata['height'], $metadata['width'] ) ) {
+			if ( isset( $metadata['height'], $metadata['width'], $metadata['length'] ) ) {
```

Comment:
I agree that it should be, plus I believe I have seen this change somewhere before already. Perhaps it was in one of the changesets of the discarded merge branch. Anyway, thanks for the pointer, I will make a separate `isset` check for the `length` property and port it back to wpcom.